joomla/joomla-cms
# [4.0] Add default stages/transitions when creating a new workflow

Pull Request for Issue #21819 .

### Summary of Changes
When creating a new workflow now all default stages (publish, unpublished, trashed, archived) and transitions (unpublish, publish, trash, archive) will be created (instead of only "published").

(And yes, transitions are not translated yet, will come after #21939)

### Testing Instructions
- Go to Content => Workflows
- Create a new workflow
- Check the stages created with the workflow
- Check the transitions created with the workflow


### Expected result
- 4 stages (unpublished, published, trashed, archived)
- 4 transitions (unpublish, publish, trash, archive)

### Actual result
- only one new stage (published)
